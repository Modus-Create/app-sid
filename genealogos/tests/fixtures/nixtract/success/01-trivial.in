{"attribute_path": "default.out", "derivation_path": "/nix/store/5r3q2vgy3951hf8vxg711zl3z2ij3fdp-trivial-1.0.drv", "output_path": "/nix/store/ys4vhl5izaahsx7mwiblsbbrlcw03w83-trivial-1.0", "outputs": [{"name": "out", "output_path": "/nix/store/ys4vhl5izaahsx7mwiblsbbrlcw03w83-trivial-1.0"}], "name": "trivial-1.0", "parsed_name": {"name": "trivial", "version": "1.0"}, "nixpkgs_metadata": {"description": "", "pname": "False", "version": "", "broken": false, "homepage": "", "licenses": null}, "src": null, "build_inputs": []}
