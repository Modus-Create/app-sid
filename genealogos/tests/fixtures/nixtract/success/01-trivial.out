{"bomFormat":"CycloneDX","components":[{"bom-ref":"/nix/store/ys4vhl5izaahsx7mwiblsbbrlcw03w83-trivial-1.0","description":"","externalReferences":[],"name":"trivial","type":"application"}],"dependencies":[{"dependsOn":[],"ref":"/nix/store/ys4vhl5izaahsx7mwiblsbbrlcw03w83-trivial-1.0"}],"serialNumber":"urn:uuid:00000000-0000-0000-0000-000000000000","specVersion":"1.5","version":1}
